--- /dev/fd/63	2026-05-18 01:51:30.688065049 -0300
+++ /dev/fd/62	2026-05-18 01:51:30.689065013 -0300
@@ -37,7 +37,7 @@
 		"charmable" 		"1" 		// Npc is charmable (1: yes, 0: no). 
 		"halo_scale"		"1.1"		// Hallow size for charmable NPCs.
 
-		"npc_health"		"5"		// NPC's life points.		
+		"npc_health"		"105"		// NPC's life points.		
 
 //		"life_tohurt"		"0.3"		// When NPC life percentage goes below this value, NPC is hurt and plays hurt animations.
 
