--- /dev/fd/63	2026-05-18 01:02:48.641864967 -0300
+++ /dev/fd/62	2026-05-18 01:02:48.642864931 -0300
@@ -27,7 +27,7 @@
 
 		"flies_spawn_time"	"10"		// Flies will be spawned after NPC is dead (time in seconds).
 		
-		"npc_health"			"25"		// NPC's life points.
+		"npc_health"			"525"		// NPC's life points.
 				
 		"maxsightdistancequiet" 	"1000"	// Maximum vision distance, in inches, when in idle mode.
 		"maxsightdistancealerted" 	"1500"	// Maximum vision distance, in inches, when in alert mode.
