--- /dev/fd/63	2026-05-18 01:21:26.124403745 -0300
+++ /dev/fd/62	2026-05-18 01:21:26.124403745 -0300
@@ -48,7 +48,7 @@
 //		"max_jump_drop"		"300"		// how high can a NPC drop. No jump for paokai.
 //		"max_jump_dist"		"200"		// how long can a NPC jump (2D). No jump for paokai.
 
-		"npc_health"		"1300"		// NPC's life points.
+		"npc_health"		"27300"		// NPC's life points.
 
 		"TimeMeleeMinAttack"	"4"		// Paokai can't attack before this time when do melee attack
 		"TimeMeleeMaxAttack"	"7"		// Paokai must attack after this time when do melee attack
