--- /dev/fd/63	2026-05-18 01:55:00.578465446 -0300
+++ /dev/fd/62	2026-05-18 01:55:00.578465446 -0300
@@ -77,7 +77,7 @@
 		"life_tohurt"			"0"  		// life percentage needed to hurt the NPC ( he plays hurt animations )
 		"stamina"			"500"		// Starting stamina
                 				
-		"npc_health"		"80"		// NPC's life points.
+		"npc_health"		"1680"		// NPC's life points.
 		
           	"mana_drainable"  	"1"		// Can be mana drained.		
 
