--- /dev/fd/63	2026-05-18 01:53:01.245786182 -0300
+++ /dev/fd/62	2026-05-18 01:53:01.245786182 -0300
@@ -18,7 +18,7 @@
 			"ThrowPrecisionMaxDist" "650.0"		// Between this distance and ThrowPrecisionMinDist, distance accuracy increase
 			"ThrowPrecisionNbShootToInc" "10.0"	// 1st shoot has very bad accuracy, ThrowPrecisionNbShootToIncrease shoot is the most accurate
 			"stamina"		"50"		// Starting stamina
-			"npc_health"		"50"
+			"npc_health"		"1050"
 		}
 		
 		"AttackCoolMinTime"		"2.0"			// Min time to beetween attacks when NPC is "cool"
@@ -104,7 +104,7 @@
 
 		"undead_creature" 	"1"		// Define is this creature is detected by the daggers of Ashes
 
-		"npc_health"		"30"		// NPC's life points.
+		"npc_health"		"1050"		// NPC's life points.
 
   		"Vulnerabilities_spells"
   		{ 
