--- /dev/fd/63	2026-05-18 01:02:52.831713263 -0300
+++ /dev/fd/62	2026-05-18 01:02:52.832713227 -0300
@@ -112,7 +112,7 @@
 		"Green_zone"			"70"		// Ideal close combat distance
 		"Red_zone"			"10"		// min close combat distance
 		
-		"npc_health"			"20"		// NPC's life points.
+		"npc_health"			"420"		// NPC's life points.
 		
 		"checkforweapons" 		"1"				// Check for weapons if it does not have one.
 		"weapon_takable_1" 		"weapon_arxorccleaver"		// Type of weapon it looks for.
