--- /dev/fd/63	2026-05-17 03:20:57.894828788 -0300
+++ /dev/fd/62	2026-05-17 03:20:57.895828808 -0300
@@ -69,15 +69,23 @@
 
 	"entity_data"
 	{
-		"WDmg"			"2"			// To be added to damage done.
-		"rangeattack"		"100"			
-		"WCritHit"		"0"			// % crithit to be added to stats
-		"WPiercing"		"0.3"			// % piercing to be added to stats
+		"WDmg"			"22"			// To be added to damage done.
+		"rangeattack"		"70"			
+		"WCritHit"		"50"			// % crithit to be added to stats
+		"WPiercing"		"0.90"			// % piercing to be added to stats 1.00=100%
+//		"magic_power" 				"1"			// Does weapon have some magical powers?
+       	"speed_attack"				"0.5"		// Animation speed in % (1.0=100%) //0.5 to consider it is unbalanced and the beak must hit. Normally used as work tool, but in combat will be difficult to use.
+		"health"           			"150"
+		"damageothercoeff" 			"1"
+		"impactothercoeff" 			"100"
+       	"crushcoeff"       			"1"
+  		"simple_weapon"  "1"			// Should be Available to parry and use skill or not. BUT doesnt work with pickaxe, why???
+       	"parry_cos"					"0.35"		// 1 : always parry. 0 : never parry.
+		"fearfactor"				"50"
 
-		"health"           	"150"
-		"damageothercoeff" 	"1"
-		"impactothercoeff" 	"100"
-       		"crushcoeff"       	"1"
-  		"simple_weapon"  "1"
+		skills_required
+		{
+			"combat"	"1"
+		}
 	}
 }
