--- /dev/fd/63	2026-05-18 01:40:07.054817684 -0300
+++ /dev/fd/62	2026-05-18 01:40:07.055817647 -0300
@@ -124,7 +124,7 @@
 		"Green_zone"			"60"		// Ideal close combat distance
 		"Red_zone"				"30"		// min close combat distance 	
 
-		"npc_health"			"20"		// NPC's life points.
+		"npc_health"			"420"		// NPC's life points.
 
 		"checkforweapons" 		"1"				// Check for weapons if it does not have one.
 		"weapon_takable_1" 		"weapon_arxclub"		// Type of weapon it looks for.
