--- /dev/fd/63	2026-05-18 01:49:05.828310056 -0300
+++ /dev/fd/62	2026-05-18 01:49:05.829310019 -0300
@@ -37,7 +37,7 @@
 		"arrow_impal" 		"1"		// Npc can be impaled by arrows (1: yes, 0: no).
 		"charmable" 		"0" 		// Npc is charmable (1: yes, 0: no). 
 		"backstable"			"1"		// Is character backstable ?
-		"npc_health"		"150"		// Npc's health.
+		"npc_health"		"63000"		// Npc's health.
 					
 		// What NPC cannot block.
 		"non_blockable2"		"arxspellmissileshot_2"		// What NPC cannot block.		
