--- /dev/fd/63	2026-05-18 01:55:24.400602906 -0300
+++ /dev/fd/62	2026-05-18 01:55:24.400602906 -0300
@@ -75,7 +75,7 @@
 
 		"flies_spawn_time"	"10"		// Flies will be spawned after NPC is dead (time in seconds).
 		
-		"npc_health"		"35"		// NPC's life points.
+		"npc_health"		"735"		// NPC's life points.
 		"base_dodge_chance"		"0.1" //0.7		// Chance to perform a dodge for defense. Doesn't stand for kick( special rules) and preventive dodge
 		"life_tohurt"			"0"  		// life percentage needed to hurt the NPC ( he plays hurt animations )
 		"stamina"			"500"		// Starting stamina
