--- /dev/fd/63	2026-05-18 01:51:02.113099676 -0300
+++ /dev/fd/62	2026-05-18 01:51:02.114099640 -0300
@@ -76,7 +76,7 @@
 
 //		"life_tohurt"			"0.3"	// When NPC life percentage goes below this value, NPC is hurt and plays hurt animations.
 				
-		"npc_health"			"180"	// NPC's life points.
+		"npc_health"			"3780"	// NPC's life points.
 
           	"mana_drainable"  		"1"	// Can be mana drained.
 
