--- /dev/fd/63	2026-05-14 02:38:10.229130538 -0300
+++ /dev/fd/62	2026-05-14 00:09:49.144305100 -0300
@@ -11,6 +11,7 @@
 "+sprint"					"#menu_controls_sprint"
 "+leanleft"					"#menu_leanleft"
 "+leanright"					"#menu_leanright"
+
 "blank"			"=========================="
 "blank"			"#menu_controls_combatmagic"
 "blank"			"=========================="
@@ -24,6 +25,7 @@
 "blank"			"#menu_controls_misc"
 "blank"			"=========================="
 "togglecharsheet" "#menu_controls_togglecharsheet"
+"+gskInvSwapPreventCrash" "#menu_controls_swapinv"
 "+use" "#menu_controls_use"
 "usebelt1" "#menu_controls_usebelt1"
 "usebelt2" "#menu_controls_usebelt2"
@@ -38,4 +40,4 @@
 "mm_nextitem" "#menu_controls_nextitem"
 "mm_objectives_show" "#menu_controls_mm_objectives_show"
 "save quick" "#Valve_Quick_Save"
-"load quick" "#Valve_Quick_Load"
\ No newline at end of file
+"load quick" "#Valve_Quick_Load"
