--- /dev/fd/63	2026-05-18 01:48:13.602201031 -0300
+++ /dev/fd/62	2026-05-18 01:48:13.602201031 -0300
@@ -147,13 +147,37 @@
 //		"TimeTestTaunt"			"4"		// Minimum time between two taunts (in second).
 
 		"checkforweapons" 		"1"				// Check for weapons if it does not have one.
-		"weapon_takable_1" 		"weapon_arx_short_sword"	// Type of weapon it looks for.
-		"weapon_takable_2" 		"weapon_arx_long_sword"		// Type of weapon it looks for.
-		"weapon_takable_3" 		"weapon_arx_silver_sword"	// Type of weapon it looks for.
-		"weapon_takable_4" 		"weapon_arx_wakizashi"		// Type of weapon it looks for.
-		"weapon_takable_5" 		"weapon_arxsword"		// Type of weapon it looks for.
-		"weapon_takable_6" 		"weapon_arxkatana"		// Type of weapon it looks for.	
-		"weapon_takable_7" 		"weapon_arxorcleaver"		// Type of weapon it looks for.
+		"weapon_takable_1" 		"weapon_arx_short_sword"
+		"weapon_takable_2" 		"weapon_arx_long_sword"
+		"weapon_takable_3" 		"weapon_arx_silver_sword"
+		"weapon_takable_4" 		"weapon_arx_wakizashi_of_excellence"
+		"weapon_takable_5" 		"weapon_arxsword"
+		"weapon_takable_6" 		"weapon_arxkatana"
+		"weapon_takable_7" 		"weapon_arxorcleaver"
+		"weapon_takable_8" 		"weapon_arxclub"
+		"weapon_takable_9" 		"weapon_mm_shield_gob"
+		"weapon_takable_10" 	"weapon_mm_weapon_generic"
+		"weapon_takable_11" 	"weapon_sword_souldrinker"
+		"weapon_takable_12" 	"weapon_arx_sword_collector"
+		"weapon_takable_13" 	"weapon_arx_sword_of_fire"
+		"weapon_takable_14" 	"weapon_mm_shield_lava"
+		"weapon_takable_15" 	"weapon_mm_shield_magic"
+		"weapon_takable_16" 	"weapon_mm_shield_necroguard"
+		"weapon_takable_17" 	"weapon_mm_shield_orc"
+		"weapon_takable_18" 	"weapon_mm_shield_deathknight"
+		"weapon_takable_19" 	"weapon_mm_hook"
+		"weapon_takable_20" 	"weapon_mm_shield_guard"
+		"weapon_takable_21"		"weapon_mm_bow_long"
+		"weapon_takable_22"		"weapon_mm_bow_plan"
+		"weapon_takable_23"		"weapon_mm_bow_sniper"
+		"weapon_takable_24"		"weapon_mm_bow_poison"
+		"weapon_takable_25"		"weapon_mm_bow_freeze"
+		"weapon_takable_26"		"weapon_mm_bow_explosive"
+		"weapon_takable_27" 	"weapon_arxcrossbow"
+		"weapon_takable_28" 	"weapon_arxorccleaver"
+		"weapon_takable_29" 	"weapon_arx_orc_chief_cleaver"
+		"weapon_takable_30" 	"weapon_arx_sword_of_misery"
+		"weapon_takable_31" 	"weapon_mm_specter_knife"
 		
 		"base_dodge_chance"		"0.5"		// Chance to perform a dodge for defense. Doesn't stand for kick( special rules) and preventive dodge
 
@@ -704,4 +728,4 @@
 		"AttackMove"	"2"
 		"range"				"40"
 	}
-}
\ No newline at end of file
+}
