--- /dev/fd/63	2026-05-18 01:30:41.659289229 -0300
+++ /dev/fd/62	2026-05-18 01:30:41.660289193 -0300
@@ -91,7 +91,7 @@
 		"non_dodgeable1"		"arxspellmissileshot_2"			// What NPC cannot dodge.
 		"non_dodgeable3"		"arxspellmissileshot"			// What NPC cannot dodge.
 							
-		"npc_health"		"75"		// NPC's life points.
+		"npc_health"		"1575"		// NPC's life points.
 
 		"flies_spawn_time"	"10"		// Flies will be spawned after NPC is dead (time in seconds).
 
