--- /dev/fd/63	2026-05-18 00:28:26.981512486 -0300
+++ /dev/fd/62	2026-05-18 00:28:26.982512450 -0300
@@ -59,7 +59,7 @@
 		"CoverChance"		"0.2"		// Chance to decide to cover in combat
 		"CoverChanceAimed"	"0.8"		// Chance to decide to cover in combat when player aim to us with spell/arrow$keyvalues
 		
-		"npc_health"		"160"		// Npc's health.
+		"npc_health"		"3360"		// Npc's health.
 
 		"min_dist_spell_defense" "100"
 		"ChanceToFreeze"	"0.5"
