--- /dev/fd/63	2026-05-18 01:22:07.517904991 -0300
+++ /dev/fd/62	2026-05-18 01:22:07.517904991 -0300
@@ -87,7 +87,7 @@
 		"move_blocker_mass"	"100"		// mass of objects NPC can push 
 		"MaxMassKickObject"	"1000"		//Max mass of object that can be kicked
 		
-		"npc_health"		"250"		// NPC's life points.
+		"npc_health"		"5250"		// NPC's life points.
 		
 		"can_be_stand_impaled"	"0"		//avoid to be directly impaled by player
 		
