--- /dev/fd/63	2026-05-18 01:47:43.433293372 -0300
+++ /dev/fd/62	2026-05-18 01:47:43.433293372 -0300
@@ -91,7 +91,7 @@
 		"maxsightdistancequiet" 	"1500"		// Maximum vision distance, in inches, when in idle mode.
 		"maxsightdistancealerted" 	"1500"		// Maximum vision distance, in inches, when in alert mode.
 
-		"npc_health"			"200"		// NPC's life points.
+		"npc_health"			"4200"		// NPC's life points.
 		"life_tohurt"			"0.1"  		// life percentage needed to hurt the NPC ( he plays hurt animations )
 		"stamina"			"50"		// Starting stamina
 		
