--- /dev/fd/63	2026-05-18 01:59:03.428672454 -0300
+++ /dev/fd/62	2026-05-18 01:59:03.428672454 -0300
@@ -72,7 +72,7 @@
 		
 		"visibilityratio"	"1.2"		// This will increase the blindness of npcs (default 1.2).
 		
-		"npc_health"		"1000"		// NPC's life points.
+		"npc_health"		"21000"		// NPC's life points.
 		
 		"Yellow_zone"		"60"		// Close combat attack only below this distance
 		"CoverTime"		"4" 		// Time NPC stay in cover
