--- /dev/fd/63	2026-05-18 01:27:50.596482981 -0300
+++ /dev/fd/62	2026-05-18 01:27:50.596482981 -0300
@@ -93,7 +93,7 @@
     		"timemaxtoaccelerate" 	"6"		// initialized to  5.f
     		"timeaccelerate" 	"4"		// initialized to 3.f (accelerate during this time)
     		
-    		"npc_health"		"75"		// NPC's life points.
+    		"npc_health"		"1575"		// NPC's life points.
 		"life_tohurt"			"0"  		// life percentage needed to hurt the NPC ( he plays hurt animations )
 		"stamina"			"500"		// Starting stamina
 
