--- /dev/fd/63	2026-06-08 01:31:23.677040652 -0300
+++ /dev/fd/62	2026-06-08 01:31:23.677040652 -0300
@@ -30,4 +30,8 @@
 fog_enable 0 
 
 //Required if using Unlimited Edition Mod:
-exec unlimitededition
\ No newline at end of file
+exec unlimitededition
+
+//The only way to grant evey cfg will be loaded in order is in this file? so better append it all here thru patches.
+//MERGE at the end of game.cfg as override
+exec eagle_eye.cfg
