--- /dev/fd/63	2026-05-18 01:02:55.651611162 -0300
+++ /dev/fd/62	2026-05-18 01:02:55.652611126 -0300
@@ -36,7 +36,7 @@
 		"angle_take_y" 		"0"		// Orientation Y.
 		"angle_take_z" 		"0"		// Orientation Z.
 
-		"npc_health"		"50"		// NPC's life points.
+		"npc_health"		"1050"		// NPC's life points.
 		
 		"maxsightdistancequiet" 	"1000"	// Maximum vision distance, in inches, when in idle mode.
 		"maxsightdistancealerted" 	"1500"	// Maximum vision distance, in inches, when in alert mode.
