--- /dev/fd/63	2026-05-18 02:03:43.736523213 -0300
+++ /dev/fd/62	2026-05-18 02:03:43.737523177 -0300
@@ -42,7 +42,7 @@
 		"fieldofviewquiet"  	"90"		// Maximum vision angle, in degres, when in idle mode.
 		"fieldofviewalerted" 	"160"		// Maximum vision angle, in degres, when in alert mode.
 
-		"npc_health"		"1000"		// Npc's health.
+		"npc_health"		"21000"		// Npc's health.
 		
 		"TimeKnockedDownByPhysics" 	"3"  	// Time npc stay knocked down after being hit by a physic.
 		
