--- /dev/fd/63	2026-05-18 01:02:51.393765328 -0300
+++ /dev/fd/62	2026-05-18 01:02:51.393765328 -0300
@@ -25,7 +25,7 @@
 		"arrow_impal" 		"0"		// Npc can be impaled by arrows (1: yes, 0: no).
 		"charmable" 		"0" 		// Npc is charmable (1: yes, 0: no).
 
-		"npc_health"		"80"		// NPC's life points.
+		"npc_health"		"1680"		// NPC's life points.
 
 		"flies_spawn_time"	"10"		// Flies will be spawned after NPC is dead (time in seconds).
 						
