--- /dev/fd/63	2026-05-18 02:03:00.308095647 -0300
+++ /dev/fd/62	2026-05-18 02:03:00.309095611 -0300
@@ -18,7 +18,7 @@
 			"ThrowPrecisionMaxDist" "650.0"		// Between this distance and ThrowPrecisionMinDist, distance accuracy increase
 			"ThrowPrecisionNbShootToInc" "10.0"	// 1st shoot has very bad accuracy, ThrowPrecisionNbShootToIncrease shoot is the most accurate
 			"stamina"		"50"		// Starting stamina
-			"npc_health"		"50"
+			"npc_health"		"1050"
 		}
 
           	"mana_drainable"  		"1"		// Can be mana drained.
@@ -104,7 +104,7 @@
 
 		"undead_creature" 	"1"		// Define is this creature is detected by the daggers of Ashes
 
-		"npc_health"		"30"		// NPC's life points.
+		"npc_health"		"1050"		// NPC's life points.
 
   		"Vulnerabilities_spells"
   		{ 
